pkg/autoscaler/autoscaler/util.go

```diff
@@ -137,6 +203,84 @@ func defaultTAC(tac *v1alpha1.TidbClusterAutoScaler) {
 	}
 }
 
+func validateBasicAutoScalerSpec(tac *v1alpha1.TidbClusterAutoScaler, spec *v1alpha1.BasicAutoScalerSpec, component v1alpha1.MemberType) error {
+	if len(spec.Rules) == 0 {
+		return fmt.Errorf("no rules defined for component %s in %s/%s", component.String(), tac.Namespace, tac.Name)
+	}
+
+	acceptableResources := map[corev1.ResourceName]struct{}{
+		corev1.ResourceCPU:     {},
+		corev1.ResourceStorage: {},
+	}
+
+	checkCommon := func(res corev1.ResourceName, rule v1alpha1.AutoRule) error {
+		if _, ok := acceptableResources[res]; !ok {
+			return fmt.Errorf("unknown resource type %s of %s in %s/%s", res.String(), component.String(), tac.Namespace, tac.Name)
+		}
+		if rule.MaxThreshold > 1.0 || rule.MaxThreshold < 0.0 {
+			return fmt.Errorf("max threshold should be between 0 and 1 for rule %s of %s in %s/%s", res, component.String(), tac.Namespace, tac.Name)
+		}
+		if len(rule.ResourceTypes) == 0 {
+			return fmt.Errorf("no resources provided for rule %s of %s in %s/%s", res, component.String(), tac.Namespace, tac.Name)
+		}
+		for _, resType := range rule.ResourceTypes {
+			if _, ok := tac.Spec.Resources[resType]; !ok {
+				return fmt.Errorf("unknown resource %s for %s in %s/%s", resType, component.String(), tac.Namespace, tac.Name)
+			}
+		}
+		return nil
+	}
+
+	for res, rule := range spec.Rules {
+		if err := checkCommon(res, rule); err != nil {
+			return err
+		}
+
+		switch res {
+		case corev1.ResourceCPU:
+			if *rule.MinThreshold > 1.0 || *rule.MinThreshold < 0.0 {
+				return fmt.Errorf("min threshold should be between 0 and 1 for rule %s of %s in %s/%s", res, component.String(), tac.Namespace, tac.Name)
+			}
+			if *rule.MinThreshold > rule.MaxThreshold {
+				return fmt.Errorf("min threshold > max threshold for cpu rule of %s in %s/%s", component.String(), tac.Namespace, tac.Name)
```

Comment:
Please also print the values of the thresholds.
Please also check the other logs.